plutus-metatheory/src/Builtin/Signature.lagda.md

```diff
@@ -1,171 +0,0 @@
----
-title: Builtin Operations Signatures
-layout: page
----
-
-This module gives a parameterised implementation of signatures of
-builtin operations. It was used in an earlier version of builtins. It
-is not used at the moment. Using it for interleaved builtins would
-require even more parameters. It is not yet clear if this would be a
-win.
```

Comment:
We used to have a more generic version of builtins that was shared in more places. I think this functionality is now copied and specialised in several different places.

It would be good to be more generic where possible but this is again not a strong enough reason to keep this file.